### PR TITLE
update make scripts to be compatiable with windows

### PR DESCRIPTION
**Pull request by tamarrow**
Requires CMake + win-bash on windows.

### CODE REVIEW

**Comment by jsancio** [comment]
@tamarrow Let's put the requirements in the README.

File: bin/packages.sh [hunk 1]
```diff
@@ -1,9 +1,17 @@
 #!/bin/bash -e
 
 BASEDIR=`dirname $0`/..
+if [ -f "$BASEDIR/env/bin/python" ]; then 
+
+	PYTHON_PATH="$BASEDIR/env/bin/python"
+else
+	PYTHON_PATH="$BASEDIR/env/Scripts/python.exe"
```
Comment (jsancio) [inline bin/packages.sh]: We should call this variable `$PYTHON`

**Comment by jsancio** [comment]
We have a few powershell scripts that we can get delete:
```
> find . -name '*.ps1'
./cli/win_bin/clean.ps1
./cli/win_bin/env.ps1
./cli/win_bin/packages.ps1
./cli/win_bin/test.ps1
./win_bin/clean.ps1
./win_bin/env.ps1
./win_bin/install/legacy/install-legacy-dcos-cli.ps1
./win_bin/install/install-dcos-cli.ps1
./win_bin/packages.ps1
./win_bin/start_tests.ps1
./win_bin/test.ps1
```

Ignore the ones under the `install` directory.

**Comment by tamarrow** [comment]
@jsancio updated!

File: README.rst [hunk 2]
```diff
@@ -48,6 +48,9 @@ Dependencies
 #. virtualenv_ must be installed and on the system path in order to install
    subcommands.
 
+#. win_bash_ must be installed if you are running this in Windows 
+   in order to run setup scripts from the Makefiles.
+
```
Comment (jsancio) [inline README.rst]: Don't we also need cmake?

**Comment by jsancio** [comment]
LGTM!

File: README.rst (hunk 2 above)
Comment (tamarrow) [inline README.rst]: nope, win-bash comes with make :)

**Pull request merged**